docs/content-services/components/search-filter-tabbed.component.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: Search filter tabbed component
+Added: v6.1.0
```

Comment:
Same here for version and date

Reply:
Updated version to 6.2.0 and Last Reviewed date to 2023-06-28